lib/phlex/bucket.rb

```
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module Phlex::Bucket
+	def const_added(name)
+		constant = const_get(name)
+
+		return if instance_methods.include?(name)
```

Perhaps should raise an error in this case.